src/components/PopoverWithoutOverlay/index.tsx

```diff
@@ -52,7 +52,7 @@ function PopoverWithoutOverlay(
                 close: onClose,
                 anchorRef,
             });
-            removeOnClose = Modal.setCloseModal(() => onClose(anchorRef));
+            removeOnClose = Modal.setCloseModal(onClose);
```

Comment:
Is this typing-related?

Reply:
`onClose` is of type `() => void`:
```ts
removeOnClose = Modal.setCloseModal(() => onClose());
// same as
removeOnClose = Modal.setCloseModal(onClose);
```

Reply:
I assume that the previous code was outdated or written in mistake? Another option would be that the typing is wrong.